src/core/server/elasticsearch/elasticsearch_service.test.ts

```diff
@@ -229,4 +268,26 @@ describe('#stop', () => {
     expect(mockAdminClusterClientInstance.close).toHaveBeenCalledTimes(1);
     expect(mockDataClusterClientInstance.close).toHaveBeenCalledTimes(1);
   });
+
+  it('stops pollEsNodeVersions even if there are active subscriptions', async () => {
+    const mockAdminClusterClientInstance = elasticsearchServiceMock.createClusterClient();
+    const mockDataClusterClientInstance = elasticsearchServiceMock.createClusterClient();
+    // @ts-ignore
```

Comment:
optional: probably we can leverage `elasticsearch.createCustomClusterClient` to operate a client with a `close` method instead of suppressing TS errors?